src/test/data/JsonSerializableMyInsuRecTest/typicalClientsMyInsuRec.json

```diff
@@ -6,48 +6,48 @@
     "phone" : "94351253",
     "email" : "alice@example.com",
     "address" : "123, Jurong West Ave 6, #08-111",
-    "tagged" : [ "friends" ]
+    "products" : [ "friends" ]
   }, {
     "@UUID" : "d4d3240e-589b-4d22-84ab-4c6bee1cca7b",
     "name" : "Benson Meier",
     "phone" : "98765432",
     "email" : "johnd@example.com",
     "address" : "311, Clementi Ave 2, #02-25",
-    "tagged" : [ "owesMoney", "friends" ]
+    "products" : [ "owesMoney", "friends" ]
```

Comment:
menial task but changing these to reflect products would be good

Reply:
this will be fixed in v1.4